docs/src/app/(public)/(content)/react/utils/use-render/demos/render/css-modules/index.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+import * as React from 'react';
+import { useRender } from '@base-ui-components/react/use-render';
+import { mergeProps } from '@base-ui-components/react/merge-props';
+import styles from './index.module.css';
+
+// ElementProps contains the 'render' prop type
+interface TextProps extends useRender.ElementProps {
+  children: React.ReactNode;
+}
+
+const Text = React.forwardRef(function Text(
```

Comment:
Not sure how we can default to React 19 and show that React 17/18 need to be different? (Of course, this works for all the versions, but is using a soon-deprecated API)

Reply:
Maybe I can revert back this change, now that we have the isntructions in the mdx.